README.md

```diff
@@ -281,6 +282,41 @@ console.log(verifier)
 console.log(challenge)
 ```
 
+### The Url format
+
+The url that the client has to visit to get a login window is `https://somtoday.nl/oauth2/authorize`.
+These are the parameters:
+
+| Name                  | Type | Value                                |
+| --------------------- | ---- | ------------------------------------ |
+| response_type         | Body | code                                 |
+| redirect_uri          | Body | [uri]                                |
+| code_challenge        | Body | [code_challenge]                     |
+| tenant_uuid           | Body | [tenant_uuid]                        |
+| oidc_iss              | Body | [oidc_iss]                           |
+| code_challenge_method | Body | [code_challenge_method]              |
+| (state)               | Body | [custom_state]                       |
+| prompt                | Body | login                                |
+| scope                 | Body | openid                               |
+| client_id             | Body | D50E0C06-32D1-4B41-A137-A9A850C892C2 |
+
+`uri` and `code_challenge` have been described already.
+`tenant_uuid` and `oidc_iss` can be found in the organisaties.json inside oidcurls
+`code_challenge_method` is the method used to encode the `code_verifier`. It is highly advised to use 'S256' wich stands for Sha256.
+`state` is an optional parameter.
+`custom_state` will be included in the callback and can be used for identification while fetching multiple tokens.
+
+After the user has logged in the page will redirect to the `uri` with these paramaters
+
+| Name           | Type | Value        |
+| ------- | ---- | ------------------- |
+| code    | Body | [code]              |
+| (state) | Body | [custom_state]      |
+| iss     | Body | https://somtoday.nl |
+
+`custom_state` is the previously defined value.
+`code` has already been described
+
 ## Fetching information
 
 baseurl: returned when you fetch a token (`somtoday_api_url`), usually [lowercase snakecased schoolname]-api.somtoday.nl
```

Comment:
```suggestion
baseurl: api.somtoday.nl
```
School names don't appear to be included in the url, at least not for my school, where `api.somtoday.nl` is enough